data/registry_model/registry_proxy_model.py

```diff
@@ -233,7 +294,7 @@ def get_repo_tag(self, repository_ref, tag_name, raise_on_error=True):
                 _, tag = self._create_and_tag_manifest(
                     repository_ref, tag_name, self._create_manifest_and_retarget_tag
                 )
-            except (UpstreamRegistryError, ManifestDoesNotExist) as e:
+            except (UpstreamRegistryError, ManifestDoesNotExist, QuotaExceededException) as e:
```

Comment:
this changes behavior compared to the pull by tag - we want to propagate quota exceptions to the end user when they happen. there's no catch `QuotaExceededException` here, it'll be propagate to the end user and treated by api exception handlers.

Reply:
removed `QuotaExceededException`